chronicnetwork/chain.json

```diff
@@ -0,0 +1,46 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "Chronic Network",
```

Comment:
must match folder name exactly, case sensitive (i.e., "chronicnetwork". You may use "Chronic Network" for the "pretty_name" property.